src/wwwroot/controllers/HeaderCtrl.js

```diff
@@ -26,6 +26,40 @@
     };
     $scope.getSubmenues = $rootScope.getSubmenues;
     $scope.isSubmenuVisible = $rootScope.isSubmenuVisible;
+
+    UpdateTrialHeader();
+    
+    function UpdateTrialHeader() {
+      $scope.isFreeTrialEnded = false;
+      $scope.isFreeTrialAlmostEnded = false;
+      $scope.isFreeTrial = false;
+      return $rootScope.freeTrialEndDate().then(function(freeTrialEndDate) {
+        var todayDate = moment().toDate();
+        var daysLeft = moment(freeTrialEndDate).diff(todayDate, "days");
+        $scope.isFreeTrial = freeTrialEndDate ? true : false;
+        $scope.trialDaysLeft = daysLeft;
+        if (freeTrialEndDate <= todayDate) {
+          $scope.isFreeTrialEnded = true;
+          $scope.isFreeTrialAlmostEnded = false;
+          $scope.showFreeTrialHeader = false;
+        }
+        if (daysLeft <= 10 && daysLeft >= 0 && freeTrialEndDate > todayDate) {
```

Comment:
Is it possible that `freeTrialEndDate > todayDate` evaluates as _true_ when `daysLeft <= 10 && daysLeft >= 0`? In what scenario does it happen?